examples/run/kokkos/seeding_example_kokkos.cpp

```diff
@@ -0,0 +1,233 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "traccc/kokkos/seeding/spacepoint_binning.hpp"
+#include "traccc/kokkos/seeding/seeding_algorithm.hpp"
+#include "traccc/device/container_h2d_copy_alg.hpp"
+#include "traccc/efficiency/seeding_performance_writer.hpp"
+#include "traccc/io/csv.hpp"
+#include "traccc/io/reader.hpp"
+#include "traccc/io/writer.hpp"
+#include "traccc/options/common_options.hpp"
+#include "traccc/options/handle_argument_errors.hpp"
+#include "traccc/options/seeding_input_options.hpp"
+#include "traccc/performance/collection_comparator.hpp"
+#include "traccc/seeding/seeding_algorithm.hpp"
+#include "traccc/seeding/track_params_estimation.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/cuda/device_memory_resource.hpp>
```

Comment:
At this point let's just set this test up to work on the host. Let's not touch CUDA specific code yet. (Also because the CI is not yet set up to allow for that.)